ci/environment-dev-3.6.yml

```diff
@@ -13,12 +13,13 @@ dependencies:
   - ipywidgets
   - jupyterlab
   # Input/Output
-  - netcdf4
+  - netcdf4=1.5.1.1
```

Comment:
Why is the version fixed? Please leave a comment for when to unfix

Reply:
This was from issues that @kpegion had. I will try going back to the original version and run all the tests. I was still never able to reproduce the error.